### PR TITLE
Ability to use jinja2 templates added.

### PR DESCRIPTION
**Pull request by sanders41**
This is in reference to issue #70. It adds the ability to use janja2 templates.

### CODE REVIEW

File: eel/__init__.py [hunk 1]
```diff
@@ -145,8 +150,12 @@ def _eel():
 
 @btl.route('/<path:path>')
 def _static(path):
-    return btl.static_file(path, root=root_path)
-
+    if _env != None:
+        template = _env.get_template(path[10:])
```
Comment (ChrisKnott) [inline eel/__init__.py]: @sanders41 What's the `10:` for here?

Comment (sanders41) [reply eel/__init__.py]: The setup assumes that templates will be placed inside a directory called "templates". The 10: removed templates/ from the path leaving only the name of the template.

Comment (ChrisKnott) [reply eel/__init__.py]: OK cheers. I will probably tweak it to make the folder name configurable

**Comment by AdamGEmerson** [comment]
Is there still a plan to merge this PR?  I came to check for this feature and lo and behold someone has already done the work!  Would love to see jinja templates integrated fully.

**Comment by ChrisKnott** [comment]
Adam - yes definitely. In fact I'll try and do this tonight.

I apologise for slow development at the moment, I haven't had much time and been focussing on better Electron support

**Comment by AdamGEmerson** [comment]
> Adam - yes definitely. In fact I'll try and do this tonight.
> 
> I apologise for slow development at the moment, I haven't had much time and been focussing on better Electron support

No worries!  I can't imagine maintaining a project like this while also balancing work, life, etc.  I'm really liking eel so far.  Thanks for your hard work!

**Comment by ChrisKnott** [comment]
@sanders41 I'm just going to make a few changes to this so that Jinja2 is not actually a dependency for people who don't need it

**Comment by ChrisKnott** [comment]
Merged. I might tidy this up in a couple of months. I am not familiar enough with Jinja2 myself at the moment to know best practices etc

**Pull request closed without merging**